src/execution/compiler/operator/analyze_translator.cpp

```diff
@@ -246,12 +257,9 @@ void AnalyzeTranslator::DeclarePgStatisticSlot(FunctionBuilder *function, ast::I
 
 void AnalyzeTranslator::DeclareAndInitPgStatisticUpdater(FunctionBuilder *function) const {
   auto *codegen = GetCodeGen();
-  // var pg_statistic_updater: StorageInterface
-  auto *storage_interface_type = codegen->BuiltinType(ast::BuiltinType::Kind::StorageInterface);
-  function->Append(codegen->DeclareVarNoInit(pg_statistic_updater_, storage_interface_type));
   // @storageInterfaceInit(pg_statistic_updater, execCtx, table_oid, pg_statistic_col_oids, true)
   auto *updater_setup = codegen->StorageInterfaceInit(
-      pg_statistic_updater_, GetExecutionContext(),
+      pg_statistic_updater_.GetPtr(codegen), GetExecutionContext(),
       catalog::postgres::PgStatistic::STATISTIC_TABLE_OID.UnderlyingValue(), pg_statistic_col_oids_, true);
```

Comment:
You'll have to `pg_statistic_col_oids_` to the pipeline state too now, otherwise it will not be found when trying to init the updater.

Reply:
oddly, it passed CI before -- do we have any analyze tests?

Reply:
ah, I think you meant with the changes. Changes have been made.